src/pydocstyle/checker.py

```diff
@@ -154,6 +183,9 @@ def check_docstring_missing(self, definition, docstring):
               with a single underscore.
 
         """
+        if is_fstring(docstring):
```

Comment:
Why is this needed?

Reply:
I think that is there to avoid getting two errors if people use an f-string as a docstring.